contractcourt/channel_arbitrator.go

```diff
@@ -132,6 +134,36 @@ type ChannelArbitratorConfig struct {
 	ChainArbitratorConfig
 }
 
+// ContractReport provides a summary of a commitment tx output.
```

Comment:
Is this only for HTLC outputs or also other types?

Reply:
It is currently just used for two types of (htlc) outputs, but the intention is to eventually use this for all commitment tx outputs.